### ISSUE TITLE
Equalizer backend feature

### ISSUE THREAD

**Issue by szuniverse** [mopidy/mopidy#1123]
Hi.

I know this feature is not so trivial... But I think it would be great if mopidy can handle somehow the sound output. I mean an equalizer backend would be nice. What do you think, is it possible somehow to support the most common sound output possibilities with a backend?

**Comment by adamcik** [comment]
http://gstreamer.freedesktop.org/data/doc/gstreamer/head/gst-plugins-good-plugins/html/gst-plugins-good-plugins-plugin-equalizer.html can do the actual work for this. And you can already statically configure this in you `audio/output` setting.

Now as for adding an API for this, that is not something I'll be looking at any time soon. Options are if #1100 can enable this, or part of the APIs for #1118 given where we are now. But this is something I feel should be postponed at least until after #1118 if it gets added at all.

**Comment by szuniverse** [comment]
okey, That's good if It will be implemented (ever). That was just a suggestion.

**Comment by ask-compu** [comment]
sorry but i cant figure out where to actually set up the settings for the gst-plugins equalizer, help?

**Comment by kingosticks** [comment]
The issue tracker here is for bugs and well-defined feature requests. Usage questions like this are better suited to http://discuss.mopidy.com, please re-post your question there.

**Comment by ask-compu** [comment]
it's alright i figured it out myself eventually
